dts/arm/nordic/nrf9160_common.dtsi

```diff
@@ -29,6 +29,13 @@ adc: adc@e000 {
 	label = "ADC_0";
 };
 
+dppi: dppi@17000 {
```

Comment:
I wonder; why didn't you name the node dppic, as its proper name (also in MDK headers)?

Reply:
changed to dppic